scripts/release.sh

```diff
@@ -37,10 +36,8 @@ if [ "$OLD_VERSION" != "none" ]; then
 	comparison="$OLD_VERSION..HEAD"
 fi
 
-if [ "$USE_RELEASE" != "1" ]; then
-	echo "Making a pre-release.."
-	pre_release="-p"
-fi
+echo "Making a pre-release.."
+pre_release="-p"
```

Comment:
this is only used once and never written to other than declaration, IMO, no variable needed

Reply:
@chrisplo: I've decided to leave it that way because we might want to add a variable to control that later. Would you like me to remove it?

Reply:
not a blocker, but would be cleaner, if the time came where it needed to be variable, it can always come back

Reply:
My concern with removing that is it's going to be lost in the flags.

Reply:
adding a comment would be a nice place to call out that it's a pre-release, or if there is a long form --pre-release flag that would be even better